### PR TITLE
Add some text and links wrt trademarks to the copyright page

### PR DESCRIPTION
**Pull request by simonpoole**
Use conventional styling fur super- and subscripts.

Added some minimal text to the copyright page wrt trademarks, currently
pointing to the LWG, one day it should refer to a formal policy.

LWG and CWG pls review. In particular I personally find the use of the TM superscript a bit ugly, but haven't come up with a better solution yet.

### CODE REVIEW

**Comment by simonpoole** [comment]
It has been suggested that ® "circle-R" might be better than superscript TM

**Comment by HolgerJeromin** [comment]
If you have a problem with the ugly html code you can use the html entity `&trade;`

File: config/locales/en.yml [hunk 1]
```diff
@@ -1078,6 +1078,9 @@ en:
         to our <a href="http://www.osmfoundation.org/wiki/License/Takedown_procedure">takedown
         procedure</a> or file directly at our
         <a href="http://dmca.openstreetmap.org/">on-line filing page</a>.
+      trademarks_title_html: <span id="trademarks"></span>Trademarks
+      trademarks_1_html: |
+        OpenStreetMap and the looking glass logo are registered trademarks of the OpenStreetMap Foundation. If you have questions about your use of the marks, please send your questions to the <a href="http://wiki.osmfoundation.org/wiki/Licensing_Working_Group">LWG</a>.
```
Comment (pnorman) [inline config/locales/en.yml]: As the first mention of the Licensing Working Group, it should be left expanded, not abbreviated to LWG.

**Comment by tomhughes** [comment]
Merged as a1f73967a929a45e32d3b517ab5918c363714c97.

**Pull request closed without merging**